chainio/clients/elcontracts/reader.go

```diff
@@ -336,3 +336,21 @@ func (r *ChainReader) CheckClaim(
 
 	return r.rewardsCoordinator.CheckClaim(&bind.CallOpts{Context: ctx}, claim)
 }
+
+func (r *ChainReader) GetOperatorAVSSplit(
+	ctx context.Context,
+	operator gethcommon.Address,
+	avs gethcommon.Address,
+) (*uint16, error) {
```

Comment:
What's the point of returning a pointer here?

Reply:
So in can be nil in the case of no value since 0 is technically valid and I want to differentiate from that and no value set.

Reply:
I'm a little confused by this. What is the return type of `GetOperatorAVSSplit`?

Reply:
`split, err := r.rewardsCoordinator.GetOperatorAVSSplit(&bind.CallOpts{Context: ctx}, operator, avs)`
does this return an error if there's not split set?

Reply:
ok pointer does not makes sense here https://github.com/Layr-Labs/eigenlayer-contracts/blob/dev/src/contracts/core/RewardsCoordinator.sol#L635-L646

if nothing is set - it is returning default. so you won't be able to differentiate